frontend/src/components/Dropdown/utils/defaultFilter.ts

```diff
@@ -23,7 +22,23 @@ export const defaultFilter = <Item extends ComboboxItem>(
       sortKeys.push('value')
     }
     sortKeys.push('description')
-    matchSorterOptions = { keys: sortKeys }
+
+    return fuzzysort
+      .go(value, items, {
+        all: true,
+        keys: sortKeys,
+        // Create a custom combined score to sort by. -500 to the desc score makes it a worse match
+        scoreFn: (a) => {
```

Comment:
i know the docs use `a` but should we rename to something like `sortKeysMatches` or something more descriptive